readthedocs/oauth/services/bitbucket.py

```diff
@@ -155,15 +165,18 @@ def create_repository(self, fields, privacy=None, organization=None):
 
             repo.html_url = fields['links']['html']['href']
             repo.vcs = fields['scm']
-            repo.account = self.account
```

Comment:
Actually, I think we talked about adding the ``remote_id`` fields later, but now that I think of it it would be better to do it now, but I'll add it in another PR after this is merged.

Reply:
Yeah. I said that in our conversation. However, I think it's better to have it _before_ we do the initial resync so we already populate this data from the beginning and then we can start making use of it. I'm fine if it's in another PR.